build.sbt

```diff
@@ -102,6 +102,7 @@ lazy val delegate = crossProject(JSPlatform, JVMPlatform)
   .in(file("delegate"))
   .settings(stdSettings("zio-macros-delegate"))
   .settings(macroSettings())
+  .settings(libraryDependencies += "dev.zio" %% "zio" % zioVersion)
```

Comment:
These are macros for `ZIO`, so I see no harm in adding `zio` dependency to the core.